curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff5850cd098f1796b5e9fa.md

```diff
@@ -0,0 +1,136 @@
+---
+id: 63ff5850cd098f1796b5e9fa
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+When you make a `GET` request to an endpoint, the data you get is not always usable at first. To make it usable, you can use the `json()` method on the response with the `.then()` method.
+
+The `.then()` method can take a callback function for the fulfilled and rejected cases of the `Promise`.
```

Comment:
I think we should add another example here for campers

```js
fetch("sample-endpoint-goes-here")
  .then((res) => res.json())
```

Reply:
Thanks. I think that would make it clearer. I've done it.